compose-files/common.yaml

```diff
@@ -1,4 +1,4 @@
-version: "3"
+version: "3.5"
```

Comment:
what features are we using from 3.5 vs 3?

Reply:
I quote a message from above:

> I completely agree that we must be coherent and all files should use the same version number. I was going to switch it to version 3, but reviewing the features per version, I see that [version 3.5](https://docs.docker.com/compose/compose-file/compose-versioning/#version-35) introduced the `name for networks, secrets and configs`, which funny enough we are using both in my new files and in your old file that use version 3. Version 3.5 requires Docker Engine version 17.09 So, I'm changing all files to version 3.5 and adding the Docker Engine minimum version to the requirements docs. If for whatever reason you still prefer v.3 we can always switch them all again, but at least they will always all have the same version.